src/components/Card.js

```diff
@@ -21,11 +26,9 @@ export default class Card {
 
   _changeLike() {
     this._cardLikeElement.classList.toggle("card__like-button_active");
```

Comment:
You need to **toggle likes** only in block `then` after a successful response from the server because if there is an error the user will not know about that and will see the toggled heart color.

Reply:
thank you, done.